storage/memory/static_test.go

```diff
@@ -99,11 +100,11 @@ func TestStaticClients(t *testing.T) {
 }
 
 func TestStaticPasswords(t *testing.T) {
-	logger := &logrus.Logger{
+	logger := log.NewLogrusLogger(&logrus.Logger{
```

Comment:
I feel like `log.NewLogrusLogger` shouldn't be necessary. *logrus.Logger should just satisfy the interface. Is there something I'm missing here?

Reply:
See my comment bellow.